src/models/index.js

```diff
@@ -126,6 +126,29 @@ module.exports = (cooler) => {
       )
       return isFollowing
     },
+    setFollowing: async ({ feedId, following }) => {
+      const ssb = await cooler.connect()
+
+      const content = {
+        type: 'contact',
+        contact: feedId,
+        following
+      }
+
+      return cooler.get(ssb.publish, content)
+    },
+    follow: async (feedId) => {
+      const isFollowing = await models.friend.isFollowing(feedId)
+      if (!isFollowing) {
+        await models.friend.setFollowing({ feedId, following: true })
```

Comment:
It would be nice if I could use `this` here, to kinda make it clearer. But I'm not sure how that works in an object literal construction.

Reply:
I could probably be persuaded, but lately I've been purposefully avoiding `this` because it reduces the amount of magic JavaScript behavior that beginners have to learn. If you'd prefer, we could turn `setFollowing()` into its own little utility function, which isn't exported to `models.friend`, but I'm quite happy with the way that you wrote it.

Reply:
yeah if you feel good about it, i'll add to the changelog and merge

Reply:
Yep, LGTM!